pkg/kclient/binding.go

```diff
@@ -121,7 +122,45 @@ func NewServiceBindingObject(bindingName string, bindAsFiles bool, deploymentNam
 	}
 }
 
-func mappingContainsBKS(bindableObjects []*meta.RESTMapping, bks sboApi.BindableKindsStatus) bool {
+// GetBindingServiceBinding returns a ServiceBinding from group binding.operators.coreos.com/v1alpha1
+func (c Client) GetBindingServiceBinding(name string) (bindingApi.ServiceBinding, error) {
+	if c.DynamicClient == nil {
+		return bindingApi.ServiceBinding{}, nil
+	}
+
+	u, err := c.GetDynamicResource(bindingApi.GroupVersionResource, name)
+	if err != nil {
+		return bindingApi.ServiceBinding{}, err
+	}
+
+	var result bindingApi.ServiceBinding
+	err = c.ConvertUnstructuredToResource(*u, &result)
+	if err != nil {
+		return bindingApi.ServiceBinding{}, err
+	}
+	return result, nil
+}
+
+// GetSpecServiceBinding returns a ServiceBinding from group servicebinding.io/v1alpha3
+func (c Client) GetSpecServiceBinding(name string) (specApi.ServiceBinding, error) {
+	if c.DynamicClient == nil {
+		return specApi.ServiceBinding{}, nil
+	}
+
+	u, err := c.GetDynamicResource(specApi.GroupVersionResource, name)
+	if err != nil {
+		return specApi.ServiceBinding{}, err
+	}
+
+	var result specApi.ServiceBinding
```

Comment:
Similar question as in `GetBindingServiceBinding` method for `var result` declaration.